lua/stacktraceplus.lua

```diff
@@ -1,3 +1,11 @@
+--[[
```

Comment:
We are using this from another project (https://github.com/ignacio/StackTracePlus IIRC), which is MIT license.